examples/srd_v1_3x2.yaml

```diff
@@ -0,0 +1,241 @@
+generate:
+  augur: # general options
+    fsky : &fsky 0.35
+    add_noise: &add_noise False # add a realization of noise?
```

Comment:
I hope that when TJPCov is more mature, firecrown will have its own syntax and interface to call TJPCov for you so you don't have to do this.